packages/salesforcedx-vscode-core/src/commands/util/apexCommandlet.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { ApexService } from '@salesforce/salesforcedx-apex/packages/apex/lib';
```

Comment:
We should be able to import library stuff by just doing `@salesforce/salesforcedx-apex`

Reply:
Once we publish the library package, we'll be able to import the library in files by just doing `@salesforce/apex`. 
But if I've understood correctly, since we're using the entire github repository, not just the library package, as the "package" in our package.json right now, we need to reference the top level `salesforcedx-apex` repository in our imports.
I haven't been able to import the library package directly using the github url, but let me know if there's something different i should try

Reply:
Or did you mean we should change the name of the repo itself to `apex`?

Reply:
I'm not sure if the github url approach supports a mono repo structure, but you are right in that once it's published we can change to `@salesforce/apex`. However, I want to make sure that we don't have to do `@salesforce/apex/lib` once we publish.

Reply:
Ah okay I see what you mean, I think we'll need to specify the `main` and `files` fields in the library's package.json so that we don't need to specify the `lib` folder. I'll open a PR in the library to address that